src/main/java/com/breskul/bibernate/persistence/PersistenceContext.java

```diff
@@ -43,12 +49,25 @@ public Set<EntityKey<?>> getEntityKeys() {
   }
 
   public <T> Object[] getEntitySnapshot(EntityKey<T> entityKey) {
+    return entitySnapshots.get(entityKey).stream()
+        .map(Pair::right)
+        .toArray();
+  }
+
+  public <T> List<Pair<String, Object>> getEntitySnapshotWithColumnName(EntityKey<T> entityKey) {
     return entitySnapshots.get(entityKey);
   }
 
-  public <T> void put(T entity) {
+  public <T> List<Triple<? extends Class<?>, String, Object>> getToOneRelationSnapshot(
```

Comment:
`Class` class is final, so no need to have here `? extends Class`

Reply:
done